backend/tracim_backend/lib/core/event.py

```diff
@@ -190,7 +206,7 @@ def create_event(
         additional_fields: typing.Dict[str, JsonDict],
         context: TracimContext,
         entity_subtype: typing.Optional[str] = None,
-    ) -> Event:
+    ) -> None:
```

Comment:
I would rather add a `return event` in the method as it seems natural to be able to get it (but you're right that it was forgotten).